src/components/texts/Body-Paragraph.tsx

```diff
@@ -1,7 +1,8 @@
 import {
   selector, useRecoilState, useRecoilValue, useSetRecoilState,
 } from 'recoil';
-
+import { useEffect, useState } from 'react';
+import { Word, Phrase } from './Phrase-Word';
```

Comment:
Remove the same line a few rows down.
~~~
2:58:05 PM: ────────────────────────────────────────────────────────────────
2:58:05 PM:   1. Build command from Netlify app                             
2:58:05 PM: ────────────────────────────────────────────────────────────────
2:58:05 PM: ​
2:58:05 PM: $ npm run build
2:58:05 PM: > frontend@0.1.0 build
2:58:05 PM: > react-scripts build
2:58:08 PM: Creating an optimized production build...
2:58:44 PM: Failed to compile.
2:58:44 PM: 
2:58:44 PM: SyntaxError: /opt/build/repo/src/components/texts/Body-Paragraph.tsx: Identifier 'Word' has already been declared. (10:9)
2:58:44 PM:    8 | } from '../../states/recoil-states';
2:58:44 PM:    9 |
2:58:44 PM: > 10 | import { Word, Phrase } from './Phrase-Word';
2:58:44 PM:      |          ^
2:58:44 PM:   11 |
2:58:44 PM:   12 | const phrasesState = selector({
2:58:44 PM:   13 |   key: 'phrasesState',
2:58:44 PM: ​
2:58:44 PM: ────────────────────────────────────────────────────────────────
2:58:44 PM:   "build.command" failed                                        
2:58:44 PM: ────────────────────────────────────────────────────────────────
2:58:44 PM: ​
~~~

Reply:
I've already done that locally. It won't compile due to this error:

```
Nav.tsx:31 Uncaught TypeError: Cannot read properties of undefined (reading 'slice')
    at capitalize (Nav.tsx:31:1)
    at children (Nav.tsx:147:1)
    at _render (render.ts:113:1)
    at render (render.ts:65:1)
    at Disclosure (disclosure.tsx:194:1)
    at renderWithHooks (react-dom.development.js:14985:1)
    at mountIndeterminateComponent (react-dom.development.js:17811:1)
    at beginWork (react-dom.development.js:19049:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:1)
capitalize @ Nav.tsx:31
children @ Nav.tsx:147
_render @ render.ts:113
render @ render.ts:65
Disclosure @ disclosure.tsx:194
renderWithHooks @ react-dom.development.js:14985
mountIndeterminateComponent @ react-dom.development.js:17811
beginWork @ react-dom.development.js:19049
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
beginWork$1 @ react-dom.development.js:23964
performUnitOfWork @ react-dom.development.js:22776
workLoopSync @ react-dom.development.js:22707
renderRootSync @ react-dom.development.js:22670
performSyncWorkOnRoot @ react-dom.development.js:22293
(anonymous) @ react-dom.development.js:11327
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushSyncCallbackQueueImpl @ react-dom.development.js:11322
flushSyncCallbackQueue @ react-dom.development.js:11309
flushPassiveEffectsImpl @ react-dom.development.js:23620
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
react_devtools_backend.js:4045 The above error occurred in the <Disclosure> component:

    at Disclosure (http://localhost:3000/static/js/bundle.js:6376:34)
    at Navbar (http://localhost:3000/static/js/bundle.js:870:91)
    at div
    at App (http://localhost:3000/static/js/bundle.js:45:81)
    at Routes (http://localhost:3000/static/js/bundle.js:48659:5)
    at Router (http://localhost:3000/static/js/bundle.js:48592:15)
    at BrowserRouter (http://localhost:3000/static/js/bundle.js:48072:5)
    at RecoilRoot_INTERNAL (http://localhost:3000/static/js/bundle.js:56790:5)
    at RecoilRoot (http://localhost:3000/static/js/bundle.js:56947:5)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
overrideMethod @ react_devtools_backend.js:4045
logCapturedError @ react-dom.development.js:20085
update.callback @ react-dom.development.js:20118
callCallback @ react-dom.development.js:12318
commitUpdateQueue @ react-dom.development.js:12339
commitLifeCycles @ react-dom.development.js:20736
commitLayoutEffects @ react-dom.development.js:23426
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
commitRootImpl @ react-dom.development.js:23151
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
commitRoot @ react-dom.development.js:22990
performSyncWorkOnRoot @ react-dom.development.js:22329
(anonymous) @ react-dom.development.js:11327
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushSyncCallbackQueueImpl @ react-dom.development.js:11322
flushSyncCallbackQueue @ react-dom.development.js:11309
flushPassiveEffectsImpl @ react-dom.development.js:23620
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
Nav.tsx:31 Uncaught TypeError: Cannot read properties of undefined (reading 'slice')
    at capitalize (Nav.tsx:31:1)
    at children (Nav.tsx:147:1)
    at _render (render.ts:113:1)
    at render (render.ts:65:1)
    at Disclosure (disclosure.tsx:194:1)
    at renderWithHooks (react-dom.development.js:14985:1)
    at mountIndeterminateComponent (react-dom.development.js:17811:1)
    at beginWork (react-dom.development.js:19049:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:1)
```